images/tiler-db/config/update_postgresql.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+if [ "${POSTGRES_DB_MAX_CONNECTIONS}X" != "X" ]; then
+  set -x;
+  sed -i -e"s/^.*max_connections =.*$/max_connections = $POSTGRES_DB_MAX_CONNECTIONS/" $PGDATA/postgresql.conf
```

Comment:
This is potentially dangerous and will have impact on memory consumption — so we should evaluate whether we add more memory or increase shmax. @Rub21 @batpad

Reply:
@geohacker  we are using this for DB tuning,  it is necessary to be there, or is there any other way that we can change those values?